### PR TITLE
Feat/color

### PR DESCRIPTION
**Pull request by edgarriba**

### CODE REVIEW

File: kornia/color/adjust.py [hunk 1]
```diff
@@ -1,42 +1,244 @@
...
+
+    The input image is expected to be an RGB image in the range of [0, 1].
 
     Args:
-        image (torch.Tensor): Image to be adjusted.
-        brightness_factor (torch.Tensor): Brightness adjust factor per element
-          in the batch. 0 generates a compleatly black image, 1 does not modify
-          the input image while any other non-negative number modify the
-          brightness by this factor.
+        image (torch.Tensor): Image to be adjusted in the shape of (*, N).
```
Comment (ducha-aiki) [inline kornia/color/adjust.py]: Don`t get the shape, actually. If it is (*, N), than it is not an image, but any nd tensor, n>1

Comment (edgarriba) [reply kornia/color/adjust.py]: I got feedback from users that might want to apply data augmentation on flatten images :D

File: kornia/color/adjust.py [hunk 2]
```diff
@@ -1,42 +1,244 @@
...
+                        f" [-0.5, 0.5]. Got {type(hue_factor)}")
+
+    # convert the rgb image to hsv
+    x_hsv: torch.Tensor = rgb_to_hsv(input)
+
+    # unpack the hsv values
+    h, s, v = torch.chunk(x_hsv, chunks=3, dim=-3)
+
+    # transform the hue value and appl module
+    h_out: torch.Tensor = (h * hue_factor) % 1
```
[the remark this answers is not in the document]
Comment (edgarriba) [reply kornia/color/adjust.py]: ha! that's what is was looking for ! thanks

**Comment by edgarriba** [comment]
@ducha-aiki had also the discussion about the saturation. In case of non standard images e.g. multispectral or depth maps that are in floating point the clamping might be an issue sometimes.

File: kornia/color/adjust.py (hunk 1 above)
Comment (ducha-aiki) [reply kornia/color/adjust.py]: I mean, just rename Image to Image/Tensor in docstring to avoid confusion

**Comment by ducha-aiki** [comment]
> @ducha-aiki had also the discussion about the saturation. In case of non standard images e.g. multispectral or depth maps that are in floating point the clamping might be an issue sometimes.

I think it is fine to merge way for only standard ones, until we know more general way.

File: kornia/color/adjust.py (hunk 1 above)
Comment (edgarriba) [reply kornia/color/adjust.py]: ah sure

**Pull request merged**